includes/commands/chungus.js

```diff
@@ -135,6 +135,7 @@ function setChungusSettings(userId, name, color) {
 }
 
 function changeName(msg, args) {
+	// this can be simplified i think?
```

Comment:
Add TODO: to the front of this, for easy searching of things to do in the future

Not a big deal at all, but since youll likely be making changes anyways, might as well.